### PR TITLE
Sameera/schooltableviewcells

### PR DESCRIPTION
**Pull request by sameerav**

### CODE REVIEW

File: sage-ios/SAGE/SAGE/SignUpTableViewController.swift [hunk 1]
```diff
@@ -112,26 +107,35 @@ class SignUpTableViewController: UITableViewController, UINavigationBarDelegate
...
+            let school = self.schools[indexPath.row]
+            var cell = self.tableView.dequeueReusableCellWithIdentifier("BrowseSchoolsCell")
+            if cell == nil {
+                cell = SchoolsTableViewCell(style: .Subtitle, reuseIdentifier: "BrowseSchoolsCell")
+            }
+            (cell as! SchoolsTableViewCell).configureWithSchool(school)
+            return cell!
+
+        } else {
+            let cell = tableView.dequeueReusableCellWithIdentifier("Cell", forIndexPath: indexPath)
```
Comment (amillman) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift:121]: is the identifier supposed to be different?

**Comment by charlesx2013** [comment]
deez commit messages tho LOL

File: sage-ios/SAGE/SAGE/SignUpTableViewController.swift (hunk 1 above)
Comment (sameerav) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift:121]: Yeah, we're using different cell classes so I wanted to keep a different identifier

**Comment by sameerav** [comment]
same

**Pull request merged**